src/test/java/edu/hm/hafner/analysis/parser/CheckStyleParserTest.java

```diff
@@ -79,6 +79,41 @@ void testParsingOfScalaStyleFormat() {
         assertThat(report).hasSize(2);
     }
 
+    /**
+     * Test parsing a file and checks the correct servity mapping for error
+     *
+     * @see <a href="https://issues.jenkins-ci.org/browse/JENKINS-56214">Issue 56214</a>
+     */
+    @Test
+    void shouldParseErrorToServityError() {
+        //Read file with one error message
```

Comment:
Do not write comments that describe the code, as these will not survive a refactoring (see [coding style](https://github.com/uhafner/codingstyle)).

Better create a private method and name the method accordingly, e.g. `readReportWithAllSeverities`